it/pages/VatReturnDetailsPageSpec.scala

```diff
@@ -84,5 +102,56 @@ class VatReturnDetailsPageSpec extends IntegrationBaseSpec {
         response.status shouldBe Status.OK
       }
     }
+
+    "the user is authenticated, has an outstanding obligation and a related " +
+      "VAT AA Return Debit Charge with an outstanding amount owed to HMRC" should {
+
+      "return 200" in new PaymentReturnRouteTest {
+        override def setupStubs(): StubMapping = {
+          AuthStub.authorised()
+          CustomerInfoStub.stubCustomerInfo
+          returnsStub.stubSuccessfulVatReturn
+          obligationsStub.stubFulfilledObligations
+          FinancialDataStub.stubAAReturnDebitChargeOutstandingPayment(5000)
+        }
+
+        val response: WSResponse = await(request().get())
+        response.status shouldBe Status.OK
+      }
+    }
+
+    "the user is authenticated, has an outstanding obligation and a related " +
```

Comment:
By the same logic as Scenario D, this test and the test below should be moved to the `/submitted/:year/:periodKey` route as a credit charge or a charge with 0 outstanding would never appear on the What You Owe page